frontends/web/src/routes/bitboxbase/bitboxbase.tsx

```diff
@@ -389,7 +420,7 @@ class BitBoxBase extends Component<Props, State> {
 
                                 <Step title="Set a Password" active={activeStep === ActiveStep.SetPassword} width={540}>
                                     <div className={stepStyle.stepContext}>
-                                        <form onSubmit={this.submitChangePassword}>
+                                        <form onSubmit={this.submitChangePasswordSetup}>
                                             <PasswordRepeatInput
                                                 label={t('initialize.input.label')}
                                                 repeatLabel={t('initialize.input.labelRepeat')}
```

Comment:
Could you set patter like https://github.com/digitalbitbox/bitbox-wallet-app/blob/e588b45d6728f3a7d4981cc837b7ac34a32b91f0/frontends/web/src/routes/device/setup/seed-create-new.jsx#L177